src/renderer/components/SetupTab/index.jsx

```diff
@@ -206,6 +214,25 @@ export default class SetupTab extends React.Component {
     writeParametersToFile(payload);
   }
 
+  async loadParametersFromFile(filepath) {
+    const datastack = await fetchDatastackFromFile(filepath);
+
+    if (datastack.module_name === this.props.pyModuleName) {
+      this.batchUpdateArgs(datastack.args);
+    } else {
+      alert(
+        `Datastack/Logfile for ${datastack.model_human_name} model does not match this model.`

```

Comment:
We have some inconsistency in the model human names where a couple of them already end in "Model", so it ends up repeating: _Datastack/Logfile for Crop Production: Percentile Model model does not match this model._ Maybe we can just leave out the second "model" there?

Reply:
Good catch. Ugh, inconsistent names. I removed the `model`.